paymentsheet/src/main/java/com/stripe/android/link/ui/FullScreenContent.kt

```diff
@@ -79,9 +92,4 @@ internal fun FullScreenContent(
             )
         }
     }
-
-    LaunchedEffect(Unit) {
-        onNavControllerCreated(navController)
-        onLinkScreenScreenCreated()
-    }
```

Comment:
Compose does not need to signal the VM to initiate rendering anymore - instead the state has the `initialDestination` bundled.